Lavanya/HTMLassignment2.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html>
+<head>
+<style>
+table, th, td {
+  border: 1px solid black;
+}
+</style>
+</head>
+<body>
+
+<h2>First Gen Pokemon Chart</h2>
+
+<table>
+  <col width="130">
+  <col width="80">
+  <col width="80">
+  <col width="80">

```

Comment:
What is **_col_** for here? **_col_** isn't HTML5. `<!DOCTYPE html>` informs the browser that it is HTML5. And isn't really necessary here in the first place.

Reply:
If we use <td style="width:150px"> the column width isn't fixed right?

Reply:
If we use td style="width:150px", the column width isn't fixed right?

Reply:
> If we use td style="width:150px", the column width isn't fixed right?

It will be. `col `is outdated and also needs to be in HTML4 or previous with the use of `colgroup`